src/hello_service.bal

```diff
@@ -5,18 +5,17 @@ import ballerina/io;
 
 // A service is a network-accessible API. This service is accessed
 // at '/hello', and bound to a listener on port 9090.
-// `http:Service`is a protocol object in the `http` package.
-service<http:Service> hello bind { port: 9090 } {
+service hello on new http:Listener(9090) {
 
-  // A resource is an invokable API method. This resource accessed
-  // at '/hello/sayHello’. `caller` is the client calling us.
-  sayHello (endpoint caller, http:Request request) {
+  //This resource accessed at '/hello/sayHello’.
```

Comment:
Dont' say "This resource" here. It assumes that the reader knows what a resource is. So instead, we have to first define a resource with something like, "A resource is an API method."

Reply:
Fix available in #45